keyboards/inland/kb83/rules.mk

```diff
@@ -1,17 +1,2 @@
-# Build Options
-#   change yes to no to disable
-#
-BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
 KEYBOARD_SHARED_EP = no
```

Comment:
Can be migrated.

Reply:
Should I migrate this as `usb.shared_endpoint.keyboard=false`, or simply delete this line?

Reply:
Didn't see it was `no`, just delete it.